debian/README.Debian

```diff
@@ -0,0 +1,18 @@
+NOTE: The Debian version of libpam-radius-auth uses as default configuration
+file /etc/pam_radius_auth.conf.
+Upstream has a default set to /etc/raddb/server that does not fit in Debian.
+Be aware that the documentation references has not been changed and they
+reflect upstream setups.
+
+Since version 2.0.0 this package ships a PAM configuration in
+/usr/share/pam-configs/radius which is turn off as default as an consequence
+of bug report #614967.
```

Comment:
Yeah, Debian-specific issues don't belong in our repo.  There are distributions using `.deb` which aren't Debian.